assembly/guide-configs/concepts/dataconversion.json

```diff
@@ -3,7 +3,7 @@
   "id": "dataconversion",
   "slug": "/dataconversion",
   "file_dir": "concepts",
-  "title": "Data conversion",
+  "title": "What is a Data Converter?",
```

Comment:
I think might need to be:
"How Temporal handles application data" or something like that.

@aarohib wdyt?